### ISSUE TITLE
Is it possoble to can move modal window?

### ISSUE THREAD

**Issue by mikeandersun** [likeastore/ngDialog#136]
Is it possible to can move modal window? Like what we see in google dialog forms.
This is a nice feature to have.
Best,

**Comment by geekygnr** [comment]
I'm sorry for re-opening this and perhaps this is my inexperience with angular showing but I am trying to combine ngDialog and angular-drag and I can not for the live of me figure out how. It should be as simple as adding a directive to the top level div but I am not seeing how.

**Comment by faceleg** [comment]
You're unlikely to find help for this sort of thing here, have you tried posting the code you have on stack overflow?

**Comment by geekygnr** [comment]
No and I am not sure I can, the project I am working on is very proprietary with NDA's and the like.

I think I found a work around. I don't know if it is correct but it works.

I just wrapped my whole template in a div gave it an absolute position and assigned the directive to that. Apparently that achieves what I want. I also have the overlay disabled. The nice thing is I can have windows but still use the controls on the base page.
